certhandler/certhandler.go

```diff
@@ -437,10 +431,10 @@ func createCSR(systemID string, extendedKeyUsage, alternativeNames []string, key
 	for _, value := range extendedKeyUsage {
 		switch strings.ToLower(value) {
 		case clientAuth:
-			oids = append(oids, oidExtKeyUsageClientAuth)
+			oids = append(oids, asn1.ObjectIdentifier{1, 3, 6, 1, 5, 5, 7, 3, 2}) // oidExtKeyUsageClientAuth
```

Comment:
Keep it as global var. It can be reused in this case. Suppress lint.

Reply:
reverted